### PR TITLE
:memo:(documentation) add cluster BBB documentation

### PR DESCRIPTION
**Pull request by NamFra**
Update documentation with a page introducing the BBB cluster

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -11,11 +11,12 @@
   * [Depuis les plateformes FUN](guide/creer-votre-classe-virtuelle/depuis-les-plateformes-fun.md)
 * [🎛 2. Configurer la classe virtuelle](guide/configurer-votre-classe-virtuelle.md)
 * [🚀 3. Lancer la classe virtuelle](guide/lancer-votre-classe-virtuelle.md)
-* [🧑‍🏫 4. Animer la classe virtuelle avec BigBlueButton](guide/animer-votre-classe-virtuelle.md)
```
Comment (sampaccoud) [inline SUMMARY.md:14]: you lost the emoji here?

Comment (NamFra) [reply SUMMARY.md:14]: Well spotted! :smile: 
In fact, unless using the GitBook app, I noticed that this emoji was persisting somewhere and consistently appearing twice (https://doc.classevirtuelle.openfun.fr/guide-dutilisation/animer-votre-classe-virtuelle)
My last option to remove it was to delete it from everywhere... :thinking:

Comment (sampaccoud) [reply SUMMARY.md:14]: :thinking:

Comment (NamFra) [reply SUMMARY.md:14]: Yep, this is odd. So much attempts to fix it... :see_no_evil:
- https://github.com/openfun/classevirtuelle.numerique-esr.fr/pull/4/commits/8e1c9568e0f2c92b2aa139bb68930094091d2605
- https://github.com/openfun/classevirtuelle.numerique-esr.fr/pull/3/commits/d3b06a6ae48f7e0c517404123a4fc2ecaced9d90
- https://github.com/openfun/classevirtuelle.numerique-esr.fr/pull/3/commits/a91d18280fd2f8e63b11712fa871de0558c2f3d3

And the emoji was still appearing in double:
image

Now it's nowhere in the .md files but it's the expected behavior :sweat_smile:

**Pull request merged**